### PR TITLE
View and Diff Commands for QC Config

### PR DESCRIPTION
**Pull request by ebelter**

### CODE REVIEW

File: lib/perl/Genome/Qc/Config.pm [hunk 1]
```diff
@@ -36,6 +37,10 @@ class Genome::Qc::Config {
     id_generator => '-uuid',
 };
 
+sub __display_name__ { sprintf('%s for %s (%s)', $_[0]->name, $_[0]->type, $_[0]->id); }
+
+sub config_to_yaml { Dump decode_json $_[0]->config; }
```
Comment (tmooney) [inline lib/perl/Genome/Qc/Config.pm]: Should `Genome::Qc::Command::Config::SaveToFile` now call this?

Comment (ebelter) [inline lib/perl/Genome/Qc/Config.pm]: I could ask :laughing:  I'll take a look!

**Comment by ebelter** [comment]
@tmooney Save config uses config_to_yaml and I added a view cmd test.

**Comment by ebelter** [comment]
CI, test this. Por favor.

**Comment by tmooney** [comment]
+1 :truck:

**Pull request merged**